README.md

```diff
@@ -4,7 +4,7 @@ Splainer Search is an Angular Solr and Elasticsearch Search library focussed on
 
 [![Build Status](https://travis-ci.org/o19s/splainer-search.svg?branch=master)](https://travis-ci.org/o19s/splainer-search)
 
-Splainer Search calls out to Solr via JSONP, asking Solr by default to try to explain and highlight all passed in fields. A friendly interface is provided to specify the Solr args in terms of a Javascript object. See below for basic examples
+Splainer Search calls out to Solr via JSONP and ElasticSearch through JSON. IT asks to explain and highlight all passed in fields. A friendly interface is provided to specify the arguments in terms of a Javascript object. See below for basic examples.

```

Comment:
small nitpicks: 
* IT asks => It asks
* technically both Solr and ES return JSON, Solr uses JSONP whereas ES uses a regular POST to make the request.